docs/operate/get-started/basics.md

```diff
@@ -0,0 +1,98 @@
+---
```

Comment:
hey as we chatted about last week - i just want to confirm this is intended to be helpful but not required to get started?  under the assumption that a lot of people very well might skip this page.  Also, i know this might not matter a huge amount, but I think i like renaming this to something without a verb since it is not procedural content.  since github uses 'about github and git' maybe 'About Viam'?  'Overview'?  etc.

Reply:
I actually kind of like the verb - feel like it encourages people to start here :P

Reply:
Oh I'd thought you wanted verbs across the board, so we came up with this and I somewhat agree with Esha though don't feel strongly. We used to have a page called "About Viam" and some people thought it meant "About Viam as a company" instead of "About the Viam platform." I do think Overview would be okay, and people would be less likely to read it.

If we *want* most people to skip over this page, then we can make it Overview, but then we need to move "What is a machine?" to each of the setup pages because we've gotten overwhelming feedback that that's a key helpful section.

Reply:
Discussed w/Esha offline and we feel that "Learn the basics" is best here. Yes people can skip over it but with this title, they're more likely to come here if they have questions as they move through other content.

Reply:
i swear I won't abuse my veto power too much but I do actually feel kind of strongly about keeping verb+object for procedural content and nouns/adjectives/etc (e.g. "About The Viam Platform" or "Viam Overview") for context-setting and/or reference content.

Reply:
For thread coherency: Ian said via email reply (see conversation page on this PR) fine to leave as-is.

Okay I will change to Viam basics in left nav.